osmtm/models.py

```diff
@@ -372,6 +376,18 @@ def to_feature(self):
             properties=properties
         )
 
+    def get_extra_instructions(self):
+        instructions = self.project.per_task_instructions
+        properties = {}
+        if self.x:
+            properties['x'] = str(self.x)
+        if self.y:
+            properties['y'] = str(self.y)
+        if self.zoom:
+            properties['z'] = str(self.zoom)
+        properties.update(_loads(self.extra_properties))
```

Comment:
Thanks for the catch here @ethan-nelson, was an oversight on my part. Have fixed this in https://github.com/hotosm/osm-tasking-manager2/pull/767/commits/14d5e170720768361051ac47c1bf1022b18832d5 and added tests for the same.